### PR TITLE
Added russian locale

### PR DESCRIPTION
**Pull request by Le6ow5k1**

### CODE REVIEW

File: config/locales/ru.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ru:
+    apipie:
+        resources: Ресурсы
+        resource: Ресурс
+        description: Описание
+        no_docs_found: Документация не найдена
+        no_docs_found_descr: Мы не нашли никакой документации для вашего АПИ.
```
Comment (IlyaBylich) [inline config/locales/ru.yml]: Probably it's better to change `АПИ` to `API`

File: config/locales/ru.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        follow_instructions_html: Перейдите по ссылке %{href}, чтобы узнать как описать ваше АПИ.
```
Comment (IlyaBylich) [inline config/locales/ru.yml]: `ваш API` instead of `ваше АПИ`

**Comment by IlyaBylich** [comment]
In general :+1:, looks good!

File: config/locales/ru.yml (hunk 1 above)
Comment (Le6ow5k1) [inline config/locales/ru.yml]: I think you right, `АПИ` looks ugly

**Comment by iNecas** [comment]
спасибо

**Pull request merged**